docs/configuration/acme.md

```diff
@@ -135,97 +133,143 @@ entryPoint = "https"
+| [Route 53](https://aws.amazon.com/route53/)            | `route53`      | `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_REGION`, `AWS_HOSTED_ZONE_ID` or a configured user/instance IAM profile. | YES                            |
+| [VULTR](https://www.vultr.com)                         | `vultr`        | `VULTR_API_KEY`                                                                                                             | Not tested yet                 |
 
-ACME certificates can be stored in a JSON file which with the `600` right mode.
+### `dnsProvider` (Deprecated)
 
-There are two ways to store ACME certificates in a file from Docker:
+!!! danger "DEPRECATED"
+    This option is deprecated. Please use [dnsChallenge.provider](/configuration/acme/#provider) instead.
+
+### `domains`
+
+You can provide SANs (alternative domains) to each main domain.
+All domains must have A/AAAA records pointing to Træfik.
+Each domain & SAN will lead to a certificate request.
 
-- create a file on your host and mount it as a volume:
-```toml
-storage = "acme.json"
-```
-```bash
-docker run -v "/my/host/acme.json:acme.json" traefik
-```
-- mount the folder containing the file as a volume
 ```toml
-storage = "/etc/traefik/acme/acme.json"
-```
-```bash
-docker run -v "/my/host/acme:/etc/traefik/acme" traefik
+[acme]
+# ...
+[[acme.domains]]
+  main = "local1.com"
+  sans = ["test1.local1.com", "test2.local1.com"]
+[[acme.domains]]
+  main = "local2.com"
+[[acme.domains]]
+  main = "*.local3.com"
+  sans = ["local3.com", "test1.test1.local3.com"]
+# ...
 ```
 
 !!! warning
-    This file cannot be shared per many instances of Træfik at the same time.
-    If you have to use Træfik cluster mode, please use [a KV Store entry](/configuration/acme/#storage-kv-entry).
+    Take note that Let's Encrypt applies [rate limiting](https://letsencrypt.org/docs/rate-limits).
 
-#### Store data in a KV store entry
+!!! note
+    Wildcard certificates can only be verified through a `DNS-01` challenge.
+
+#### Wildcard Domains
 
-ACME certificates can be stored in a KV Store entry.
+[ACME V2](https://community.letsencrypt.org/t/acme-v2-and-wildcard-certificate-support-is-live/55579) allows wildcard certificate support.
+As described in [Let's Encrypt's post](https://community.letsencrypt.org/t/staging-endpoint-for-acme-v2/49605) wildcard certificates can only be generated through a [`DNS-01` challenge](/configuration/acme/#dnschallenge).
 
 ```toml
-storage = "traefik/acme/account"
+[acme]
+# ...
+[[acme.domains]]
+  main = "*.local1.com"
+  sans = ["local1.com"]
+# ...
 ```
 
-**This kind of storage is mandatory in cluster mode.**
-
-Because KV stores (like Consul) have limited entries size, the certificates list is compressed before to be set in a KV store entry.
-
-!!! note
-    It's possible to store up to approximately 100 ACME certificates in Consul.
+It is impossible possible to define a wildcard on a wildcard domain (for example `*.*.local.com`).
```

Comment:
`It is not possible to request a double wildcard certificate for a domain (for example `*.*.local.com`)`